constructor/nsis/main.nsi.tmpl

```diff
@@ -803,7 +811,12 @@ Section "Install"
     @PKG_COMMANDS@
 
     # Cleanup
+    SetOutPath "$INSTDIR"
     Delete "$INSTDIR\pkgs\env.txt"
+    ${If} $ARGV_KeepPkgCache == "0"
+        DetailPrint "Clearing the package cache..."
+        nsExec::ExecToLog '"$INSTDIR\_conda.exe" clean --all --force-pkgs-dirs --yes'
```

Comment:
This changed my mind: `conda` can do a better job removing the package cache than NSIS can. So it's reasonably to correct the default behavior to remove the package cache.